scripts/compare_against_released_templates_test.py

```diff
@@ -133,6 +133,14 @@ def _support_compare_index_templates(self, args, json_url):
         if 'aushape' in released_data["mappings"]["_default_"]["properties"]:
             released_data["mappings"]["_default_"]["properties"]["aushape"]["properties"]["error"]["index"] = "analyzed"
             released_data["mappings"]["_default_"]["properties"]["aushape"]["properties"]["error"]["doc_values"] = False
+
+        # We need to remove the `include_in_all` from downloaded model because this setting has been was removed.
```

Comment:
There is an error at the end of the line.

Reply:
Fixed.